tests/vmi_multiqueue_test.go

```diff
@@ -93,16 +95,13 @@ var _ = Describe("[sig-compute]MultiQueue", decorators.SigCompute, func() {
 		)
 
 		It("[test_id:959][rfe_id:2065] Should honor multiQueue requests", func() {
-			vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskAlpine))
-			Expect(numCpus).To(BeNumerically("<=", availableCPUs),
+			Expect(availableCPUs).To(BeNumerically(">=", numCpus),
 				fmt.Sprintf("Testing environment only has nodes with %d CPUs available, but required are %d CPUs", availableCPUs, numCpus),
 			)
 
+			cpuResources := strconv.Itoa(int(numCpus))
```

Comment:
Why is it better than the original `fmt.Sprintf("%d", numCpus)` ?

Reply:
I bet the performance is negligible; and I have no certainty what would be better. In any case, I don't think that efficiency is the subject of this comment. I wouldn't touch it in this PR.

Reply:
for this code:
```go
package fmtvsconv

import (
	"fmt"
	"strconv"
	"testing"
)

func BenchmarkFMT(B *testing.B) {
	for i := 0; i < B.N; i++ {
		_ = fmt.Sprintf("%d", i)
	}
}

func BenchmarkConv(B *testing.B) {
	for i := 0; i < B.N; i++ {
		_ = strconv.Itoa(i)
	}
}
```

when running, the results are very clear:
```shell
$ go test -bench=.
goos: linux
goarch: amd64
pkg: github.com/nunnatsa/workspace/fmtvsconv
cpu: 11th Gen Intel(R) Core(TM) i7-11850H @ 2.50GHz
BenchmarkFMT-16         22089375                56.51 ns/op
BenchmarkConv-16        61178786                18.88 ns/op
PASS
ok      github.com/nunnatsa/workspace/fmtvsconv 2.483s
```

Reply:
> I bet the performance is negligible; and I have no certainty what would be better. In any case, I don't think that efficiency is the subject of this comment. I wouldn't touch it in this PR.

As I see it, this is a tiny change, in a function we're moving and changing anyway. The change is between two standard lib functions, so it's completely safe - it's not like changing an algorithm or something like that.

Reply:
As a reviewer, I find this change as a distraction. It has nothing to do with the purpose of the PR. It shaves 40ns from a test lane that runs for hours. In my opinion, it does not belong here.

Reply:
This is a classic case where the reviewer has priority and it does not matter who is right.

I do not see the point fighting over this now, but please reconsider this for the next changes.

Reply:
I totally disagree. IMO code should be always written in an efficient manner, no matter in which context the code is running, be it a part of test suite that runs for decades, or a code that runs on a satellite. Especially when its a small change.